.travis.yml

```diff
@@ -1,11 +1,21 @@
 language: python
-python:
-  - "3.4"
-  - "3.5"
-  - "3.6"
+cache: pip
+git:
+  depth: 20
+cache:
```

Comment:
There should additionally be `pip: true` in `cache:` so that pipenv itself doesn't have to be redownloaded each time.

Reply:
Is that since we should combine the `cache:` entry, and when we do it needs `pip: true`?

Perhaps like this?
https://docs.travis-ci.com/user/caching/#enabling-multiple-caching-features

Reply:
@rht Is this what you meant? ^

Reply:
Yes, to be exact:
```
cache:
  pip: true
  directories:
    - $HOME/.cache/pipenv
```
Otherwise I'm not sure how yaml handles a field redefinition.